.github/workflows/deployment.yaml

```
@@ -78,21 +78,35 @@ jobs:
         env:
           ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
           ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
-        run: curl --fail --user $ARTIFACTORY_USERNAME:$ARTIFACTORY_PASSWORD --upload-file dist/linux-64/jdk4py-*.tar.bz2 "$JDK4PY_CONDA_CHANNEL/linux-64/"
+        run: |
+          ls dist/linux-64/jdk4py-*.tar.bz2 | \
+          while read f; do
```

Make it a bit easier to read:

```suggestion
          while read filepath; do
```